src/main.py

```diff
@@ -1,188 +1,100 @@
+            responded_to.append(sender.id)
+        elif 'for_others' in config['messages']:
+            await event.reply(config['messages']['for_others'] + config['messages']['footer'], link_preview=False)
+            responded_to.append(sender.id)
 
+        times_responded += 1
 
-	atexit.register(cronitor_atexit)
+        if config['service']['logging_enabled']:
+            log_response(sender, event.message.text)
 
+        if config['service']['cronitor_integrated']:
+            cronitor.inform(sender, event.message.text, times_responded)
 
-@client.on(events.NewMessage(incoming=True))
-async def handler(event):
-	global times_responded
-	if type(event.peer_id) is not PeerUser:
-		return
-	try:
-		sender = await client.get_entity(event.peer_id)
-	except:
-		return
-	try:
-		# FIXME:
-		# Dumbass approach
-		known_phones = []
-		known_ids = []
-		known_usernames = []
-		for i in config['contacts']['known']:
-			if i[0] == '@':
-				known_usernames.append(i[1:])
-			if i[0] == '+':
-				known_phones.append(i[1:])
-			else:
-				known_ids.append(i)
-		ignore_phones = []
-		ignore_ids = []
-		ignore_usernames = []
-		for i in config['contacts']['ignore']:
-			if i[0] == '@':
-				ignore_usernames.append(i[1:])
-			if i[0] == '+':
-				ignore_phones.append(i[1:])
-			else:
-				ignore_ids.append(i)
-		if (sender.is_self
-				or sender.bot
-				or sender.support
-				or str(sender.id) in ignore_ids
-				or sender.username in ignore_usernames
-				or sender.phone in ignore_phones
-				or (config['messages']['respond_only_once'] and sender.id in responded_to)):
-			return
-		if 'for_known' in config['messages'] and (
-				str(sender.id) in known_ids or sender.username in known_usernames or sender.phone in known_phones):
-			await event.reply(config['messages']['for_known'] + FOOTER, link_preview=False)
-			responded_to.append(sender.id)
-		elif 'for_others' in config['messages']:
-			await event.reply(config['messages']['for_others'] + FOOTER, link_preview=False)
-			responded_to.append(sender.id)
-		times_responded += 1
-		if config['service']['logging_enabled']:
-			log_response(sender, event.message.text)
-		if config['service']['cronitor_integrated']:
-			cronitor_inform(sender, event.message.text, times_responded)
-	except Exception as e:
-		my_except_hook(type(e), e)
+    except Exception as e:
+        sys.excepthook(type(e), e)
 
 
 client.connect()
 if not client.is_user_authorized() and not setup_mode:
-	raise PermissionError("You are unauthorized. Retry with '-s' or '--setup' to authorize")
+    raise PermissionError("You are unauthorized. Retry with '-s' or '--setup' to authorize")
+
 log_str = f"Starting Pong v{VERSION}..."
 print(log_str)
+
 if config['service']['logging_enabled']:
-	logging.info(log_str)
+    logging.info(log_str)
+
 if config['service']['cronitor_integrated']:
-	cronitor_informstate(log_str, "run")
-del CONFIG_FILE_NAME, VERSION, setup_mode, log_str
```

Comment:
Why removed

Reply:
Useless

Reply:
I mean :183

Reply:
Still useless. Python takes care of memory by itself